covid_xprize/scoring/prescriptor_scoring.py

```diff
@@ -0,0 +1,164 @@
+import os
+
+import pandas as pd
+
+from covid_xprize.standard_predictor.predict import predict
+from covid_xprize.standard_predictor.xprize_predictor import NPI_COLUMNS
+
+
+def weight_prescriptions_by_cost(pres_df, cost_df):
+    """
+    Weight prescriptions by their costs.
+    """
+    weighted_df = pres_df.merge(cost_df, how='outer', on=['CountryName', 'RegionName'], suffixes=('_pres', '_cost'))
+    for npi_col in NPI_COLUMNS:
```

Comment:
There's probably some fancy `numpy` vectorized "dot product" invocation to do this. Minor optimization of course.